snuba/migrations/snuba_migrations/metrics/0030_metrics_distributions_v2_writing_mv.py

```diff
@@ -39,7 +39,7 @@ def forwards_local(self) -> Sequence[operations.SqlOperation]:
                 ),
                 metric_type="distribution",
                 target_mat_version=4,
-                appended_where_clause="AND timestamp > toDateTime('2022-03-29 00:00:00')",
+                appended_where_clause="AND timestamp >= toDateTime('2022-03-22 00:00:00')",
```

Comment:
Why not just removing it. 
Also I think you will have the same issue with sets

Reply:
> I am not sure this will fix the problem though. Since the migration already landed in master and it was applied on everybody's developer environment, changing the timestamp will not have any impact for those that already applied the migration. To fix that you will have to add a new migration that creates a new matview and drops the existing.

yeah, I'm doing this now